apps/notification/views.py

```diff
@@ -71,3 +71,34 @@ def get_count(self, request, version=None):
             'total': total,
         }
         return response.Response(result)
+
+
+class AssignmentViewSet(viewsets.ModelViewSet):
+    serializer_class = AssignmentSerializer
+    permission_classes = [permissions.IsAuthenticated]
+    filterset_fields = ('project',)
+
+    def get_queryset(self):
+        return Assignment.get_for(
+            self.request.user,
+        )
```

Comment:
@AdityaKhatri @samshara Do we need to filter out the already done assignments or client will request with a filter?

Reply:
Best to have the client side do it, in case we need the list of already done assignments in future.

Reply:
@k9845 Let's add the is_done filter in the filterset too.

Reply:
@thenav56 
done